lita_config.rb.example

```diff
@@ -1,5 +1,3 @@
-require "lita-slack"
```

Comment:
So you actually need this bit if you are setting up slack integration.  Maybe comment it out saying "uncomment once you put in slack tokens below"